### ISSUE TITLE
A11y - Fancy help menus should have tab-able buttons

### ISSUE THREAD

**Issue by showeimer** [IBM/kui#3077]
Command links should be able to be reached via tab and receive focus:

<img width="596" alt="Screen Shot 2019-10-22 at 3 15 47 PM" src="https://user-images.githubusercontent.com/26075187/67322339-e0074280-f4de-11e9-8268-1057ece0a16c.png">
